BuildScripts/mvSimpleFunctions.sql

```diff
@@ -1953,6 +2009,52 @@ LANGUAGE    plpgsql
 SECURITY    DEFINER;
 ------------------------------------------------------------------------------------------------------------------------------------
 CREATE OR REPLACE
+FUNCTION    mv$removeIndexFromMv$Table
+            (
+                pConst          IN      mv$allConstants,
+                pIndexName      IN      TEXT
+            )
+    RETURNS VOID
+AS
+$BODY$
+/* ---------------------------------------------------------------------------------------------------------------------------------
+Routine Name: mv$removeIndexFromMv$Table
+Author:       Rohan Port
+Date:         15/06/2021
+------------+---------------+-------------------------------------------------------------------------------------------------------
+Description:    This function removes an index from the materilized view table
+
+Arguments:      IN      pConst              The memory structure containing all constants
+                IN      pIndexName          The name of the index to remove
+Returns:                VOID
+************************************************************************************************************************************
+Copyright 2021 Beyond Essential Systems Pty Ltd
```

Comment:
I have no idea how copyright law works... the rest of these functions have an individual copyright declaration that includes `Amazon` in it, but it fully open source

I don't think it really matter but just wanted to flag this in case I'm committing crimes somehow...

Reply:
Yeah not sure either, let's get legal onto it! Jokes...we have no legal. I think it's fine.